src/main/resources/view/MainWindow.fxml

```diff
@@ -43,9 +43,14 @@
                  <tabs>
                    <Tab text="Home">
                         <content>
-                      <StackPane fx:id="personListPanelPlaceholder" />
+                      <StackPane fx:id="taskListPanelPlaceholder" />
                         </content>
                    </Tab>
+                   <Tab text="AddressBook">
```

Comment:
There is a separate tab for Supplier with a placeholder. I think we can just keep this tab as AddressBook first (since it is technically not supplier right now, so we have a tab for the original AddressBook and a tab for suppliers) so as to avoid confusion